decidim-forms/app/presenters/decidim/forms/answer_option_presenter.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Forms
+    #
+    # Decorator for answer_options
+    #
+    class AnswerOptionPresenter < SimpleDelegator
+      include Decidim::TranslationsHelper
+
+      def translated_body
+        @translated_body ||= translated_attribute body
+      end
+
+      def json
```

Comment:
I would rather use the standards :to_json or :as_json
```suggestion
      def as_json
```